source/partials/_prizes.html.erb

```diff
@@ -0,0 +1,35 @@
+<section class="prizes">
+  <h2 class="headline">Prizes</h2>
+  <p class="main-description">
+    There will be a jury to decide which project is the best.
+  </p>
+  <p class="main-description">
+    The three judging criteria will be:
+  </p>
+  <dl>
+    <dt>The Impact</dt><dd>how much change the project can make.</dd>
+    <dt>The Pitch</dt><dd>do you tell a compelling story?</dd>
+    <dt>The Demo</dt><dd>we want to experience your vision</dd>
```

Comment:
It might be nicer to have a consistent format here. the first one is an indirect question with period, the second a question with question mark, the third a statement with no punctuation. just saying…